messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -696,6 +700,12 @@ private CompletableFuture<Void> awaitTermination() {
                                     if (workerThread.isAlive()) {
                                         workerThread.interrupt();
                                         workerThread.join(workerTerminationTimeout);
+                                        if (workerThread.isAlive( )) {
+                                        	logger.warn(
+                                        			"Forced shutdown of TrackingProcessor Worker '{}' was not successful. Consider increasing workerTerminationTimeout.",
```

Comment:
The message seems fair and helpful, thanks Nils! I just have a small caveat here: 

```suggestion
                                        			"Forced shutdown of Tracking Processor Worker '{}' was unsuccessful. Consider increasing workerTerminationTimeout.",
```

The only other point that crosses my mind, is that there might also be an issue with the user's Event Handlers or the services these handlers use/invoke.
Perhaps adding another sentence that revisiting the event handler implementation wouldn't hurt either.